cmd/manager/main.go

```diff
@@ -75,8 +75,8 @@ func main() {
 	var logAPIRequests bool
 	pflag.BoolVar(&logAPIRequests, "logAPIRequests", false, "Log API requests performed by the operator.")
 
-	var runLocalBuild bool
-	pflag.BoolVar(&runLocalBuild, "runLocalBuild", false, "Run the operator locally.")
+	var leaderElect bool
+	pflag.BoolVar(&leaderElect, "leader-elect", true, "Enable a leader election option in the operator manager.")
```

Comment:
```suggestion
	pflag.BoolVar(&leaderElect, "leader-elect", true, "Enable leader election for this operator. Enabling this will ensure there is only one active controller manager.")
```